pkg/occlient/occlient.go

```diff
@@ -1157,16 +1180,45 @@ func (c *Client) GetBuildConfig(name string, project string) (*buildv1.BuildConf
 	return bc, nil
 }
 
-// GetClusterServiceClasses queries the service service catalog to get all the
-// currently available cluster service classes
+// GetClusterServiceClasses queries the service service catalog to get available clusterServiceClasses
 func (c *Client) GetClusterServiceClasses() ([]scv1beta1.ClusterServiceClass, error) {
-	classList, err := c.serviceCatalogClient.ClusterServiceClasses().List(metav1.ListOptions{})
+	// TODO: Remove `FieldSelector` from ListOptions when we are confident with Ansible Service Broker
```

Comment:
Why are we not confident with ASB?

Reply:
because ASB is highly parameterised and issue talks about templates only

Reply:
issue #266

Reply:
@surajnarwade Can you please add a comment explaining why are we using just template service broker?